src/__internal__/character-count/character-count.component.tsx

```diff
@@ -14,14 +15,25 @@ const CharacterCount = ({
   limit,
   isOverLimit,
   "data-element": dataElement,
-}: CharacterCountProps) => (
-  <StyledCharacterCount
-    aria-live="polite"
-    isOverLimit={isOverLimit}
-    data-element={dataElement}
-  >
-    {`${value}/${limit}`}
-  </StyledCharacterCount>
-);
+}: CharacterCountProps) => {
+  const remainingCharacters: number = +limit - +value;
+  const charactersOverLimit: number = +value - +limit;
+  const l = useLocale();
+  const remainingCharactersString = l.characterCount.remainingCharactersString();
```

Comment:
why this is only in plural?

Reply:
Great point, I've spent most of today finding a better way to achieve this automatically, and with the use of translations